packages/common/src/audius-query/hooks/usePaginatedQuery.ts

```diff
@@ -80,7 +81,9 @@ export const useAllPaginatedQuery = <
       if (!statusIsNotFinalized(result.status)) {
         setLoadingMore(false)
       }
+      setStatus(result.status)
```

Comment:
cc @amendelsohn. This _seems_ like a safe change, but would appreciate your review here. It's not the perfect long term solution, but it does prevent the status getting set ahead of the data. 😅

Reply:
Haha uhhhh yeah that seems fine ✅